src/abstractLocator.ts

```diff
@@ -226,4 +213,59 @@ export abstract class AbstractLocator {
         return cacheFile;
     }
 
+    private refreshConfig(): boolean {
+        const config = vscode.workspace.getConfiguration("projectManager");
+        let refreshedSomething: boolean = false;
+        let currentValue = null;
+
+        currentValue = config.get<string[]>(this.getKind() + ".baseFolders");
+        if (!this.arraysAreEquals(this.baseFolders, currentValue)) {
+            this.baseFolders = currentValue;
+            refreshedSomething = true;
+        }
+
+        currentValue = config.get<string[]>(this.getKind() + ".ignoredFolders", []);
+        if (!this.arraysAreEquals(this.baseFolders, currentValue)) {
```

Comment:
At this point, the `currentValue` should be checked against `this.ignoredFolders`, not `this.baseFolder`. I guest it was just a _copy/paste_ issue.